sklearn/metrics/_plot/tests/test_plot_roc_curve.py

```diff
@@ -131,4 +131,28 @@ def test_roc_curve_not_fitted_errors(pyplot, data_binary, clf):
         plot_roc_curve(clf, X, y)
     clf.fit(X, y)
     disp = plot_roc_curve(clf, X, y)
+    assert clf.__class__.__name__ in disp.line_.get_label()
```

Comment:
```suggestion
    assert clf.__class__.__name__ == disp.line_.get_label()
```

Reply:
no because the docstring as the AUC string as well

Reply:
and I am not sure that a regex would be simpler to understand :)

Reply:
whoooops. (I forgot)